src/duration/mod.rs

```diff
@@ -178,19 +217,61 @@ impl SubAssign for Duration {
     }
 }
 
+impl Mul<u32> for Duration {
+    type Output = Self;
+
+    fn mul(self, rhs: u32) -> Duration {
+        Self(self.0 * rhs)
+    }
+}
+
+impl Mul<Duration> for u32 {
+    type Output = Duration;
+
+    fn mul(self, rhs: Duration) -> Duration {
+        rhs * self
+    }
+}
+
+impl Div<u32> for Duration {
+    type Output = Self;
+
+    fn div(self, rhs: u32) -> Duration {
+        Self(self.0 / rhs)
+    }
+}
+
+impl Div<Duration> for u32 {
+    type Output = Duration;
+
+    fn div(self, rhs: Duration) -> Duration {
+        rhs / self
+    }
+}
```

Comment:
Division is not symmetric (`x / y != y / x`).

Reply:
Yes, you are right. We should remove `impl Div<Duration> for u32`, because:

1. the implementation is wrong
2. doesn't really make sense (why would you divide a `u32` by `Duration`)

Reply:
Removed in 9a5c6da